src/SFA.DAS.RoATPService.Application/Handlers/CreateOrganisationHandler.cs

```diff
@@ -99,7 +99,7 @@ public class CreateOrganisationHandler : IRequestHandler<CreateOrganisationReque
             _logger.LogInformation($@"Handling Create Organisation Search for UKPRN [{request.Ukprn}]");
   
             var command = _mapper.Map(request);
-
+            command.SourceIsUKRLP = true;
```

Comment:
I think this field should be mapped over from the request, as the API may be used by other apps in future